app/domain/authentication/authn_jwt/identity_providers/identity_from_decoded_token_provider.rb

```diff
@@ -28,14 +28,14 @@ def jwt_identity
         end
 
         # Checks if variable that defined from which field in decoded token to get the id is configured
-        def identity_available?
-          identity_field_variable.present?
+        def identity_available
+          @identity_available ||= identity_field_variable.present?
```

Comment:
fix memoization to new memoization (with unless)
In the current memoization if `identity_field_variable.present?` equals `false`, `identity_field_variable.present?` will be called again (it will not be memoized)